static/app/components/modals/inviteMembersModal/inviteStatusMessage.tsx

```diff
@@ -57,12 +94,45 @@ export default function InviteStatusMessage({
       );
     }
     const inviteRequests = (
-      <strong>{tn('%s invite request', '%s invite requests', sentCount)}</strong>
+      <strong data-test-id="sent-invite-requests">
+        {tn('%s invite request', '%s invite requests', sentCount)}
+      </strong>
+    );
+    const failedInviteRequests = (
+      <strong data-test-id="failed-invite-requests">
+        {tn('%s invite request', '%s invite requests', errorCount)}
+      </strong>
     );
     const tctComponents = {
       inviteRequests,
       failed: errorCount,
+      failedInviteRequests,
```

Comment:
🤔 is there a difference other than the test id between this and the `failedInvites` component?

Reply:
no 😞 fixed